### ISSUE TITLE
Dockerfile git address for Synchronized-BatchNorm-PyTorch missing ".git"

### ISSUE THREAD

**Issue by sammonius** [mcheng89/gaugan#6]
it doesn't clone cuz it needs ".git" at the end.
